operator/v1/types_console.go

```diff
@@ -174,6 +178,15 @@ type DeveloperConsoleCatalogCategoryMeta struct {
 	Tags []string `json:"tags,omitempty"`
 }
 
+type AccessReviewResources struct {
+	// kind is a string value representing the REST resource.
+	// +required
+	Kind string `json:"kind"`
+	// verb of API request (get, list, create, etc...).
+	// +required
+	Verb string `json:"verb"`
```

Comment:
Are verbs required to be lower case? Can you only set a single verb or should this be a verbs list?